macros/log_to_latest_nested.sql

```diff
@@ -0,0 +1,14 @@
+{%- macro log_to_latest_nested(table, key, sort_key) -%}
+  SELECT
+  deduped.* EXCEPT(row_num)
+FROM (
+  SELECT
+    _inner.*,
+    ROW_NUMBER() OVER (PARTITION BY {{ key }} ORDER BY {{ sort_key }} DESC, (SELECT a.value FROM UNNEST(_attributes) AS a
+      WHERE
+        a.key = 'timestamp' ) DESC ) AS row_num
```

Comment:
This could be generalised more but fine to leave it for future PR. Thinking if "timestamp" could be something else like "datetime." I think it's fare to assume that we have a `value/key` pair, but the `_attributes` column could also be named differently 🤔

Reply:
Yes good point! I can update it